bridget/eddy/footprint.py

```diff
@@ -1,3 +1,5 @@
+import numpy as np
+
```

Comment:
In Python, place the import always after the first comment. The first comment is always the documentation for the whole file. e.g.:

```python
"""
Description of the whole file/module. 

- what is a footprint
- how to import
- how to use
"""
import numpy as np


def my_first_function():
    """
    documentation of the first function.
    """
    ...
```

You don't have to perfectly fill all the docstrings with content.